Cargo.lock

```diff
@@ -68,16 +68,16 @@ version = "0.7.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "fcb51a0695d8f838b1ee009b3fbf66bda078cd64590202a864a8f3e8c4315c47"
 dependencies = [
- "getrandom 0.2.3",
+ "getrandom 0.2.8",
  "once_cell",
  "version_check",
 ]
 
 [[package]]
 name = "aho-corasick"
-version = "0.7.18"
+version = "0.7.20"
```

Comment:
We prefer not to have `cargo update` and review individual dependabot updates.

Reply:
Ah oops, the `cargo upgrade` command wasn't working so I tried `cargo update`. I always get those mixed up, quite confusing IMO.